### DISCUSSION TITLE
반성합니다 ㅠㅠ

### DISCUSSION THREAD

**Discussion by pykido** [BDD-CLUB/challenge-every-week-posting#605] [6. 반성문]
반성합니다 ㅠㅠ... 저도 지민님 따라서 경단 프로젝트한다고 정신이 너무 없었네요.. 
그래도 결과물이 만족스러운 거 같아 너무 기분이 좋습니다!
개강 전에 목표를 다시 세웠는데 수행하는 과정을 앞으로 블로그에 꾸준히 업로드해볼게요! 다들 화이팅!!!

**Comment by 02ggang9** [comment]
화이팅입니다~~

**Reply by pykido** [reply]
감사합니다!!!! ㅎㅎㅎ 수빈님도 화이팅입니다.

**Comment by stopmin** [comment]
굿굿~~ 경단 회고도 써주세요 ㅎㅎ

**Reply by pykido** [reply]
넵넵
